src/main/java/core/mate/academy/model/ExcavatorProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    @Override
+    public List<? extends Machine> get() {
```

Comment:
The method `get` should return `List<Excavator>` instead of using a wildcard type `List<? extends Machine>`. This ensures that the list is specifically typed to `Excavator`, which is more precise.